composer/2022_airflow_summit/summit_dag_process.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import sys
+
+
+from py4j.protocol import Py4JJavaError
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import UserDefinedFunction
+from pyspark.sql.types import FloatType
+
+
+def temperature_celsius_udf(temperature_tenths):
+    """Convert temperature from tenths of a degree in celsius to degrees celsius"""
+    if temperature_tenths:
+        return temperature_tenths / 10
```

Comment:
Not sure I follow what the intention of this UDF is based on the docstring / logic.

Reply:
LOL at my arithmetic mistake. The dataset has a column with temperature in 10ths of a degree celsius and I needed to be multiplying not dividing by 10 to get actual degrees celsius.